_posts/2016-12-08-underhanded-rust.en.md

```diff
@@ -0,0 +1,212 @@
+---
+layout: post
+title:  "The 2016 Underhanded Rust Contest"
+date:   2016-12-07 11:23:34 -1000
+categories: underhanded
+lang: en-US
+---
+
+# Introduction
+
+The [Rust Community Team](https://community.rs) is pleased to announce the
+first annual Underhanded Rust Contest, a competition that tests our assertion
+that [Rust](https://www.rust-lang.org/), and our
+[ecosystem](https://crates.io/), is easier to write clean and robust software.
```

Comment:
This is a pretty complicated sentence. I think there should be an _in_ at the end, or at least change "is easier" to "make it easier". As a reader I'm immediately asking "Easier compared to what?". Can you maybe qualify that comparison[^1] or just say "easy".

[^1]: like "easier compared to legacy C++", whatever "legacy C++" actually is